### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by rtsisyk**
Signed-off-by: Roman Tsisyk <roman@tsisyk.com>

##

[strings] add missing translations for craft=*
[styles] Add driving_school, music_school and language_school
[editor] Fixed exception type.
[search] Better street's FeaturesLayer filtering logic.
[search] Decrease score for numeric streets like "2-й ...".
[search] Increased penalty for error; increased city rank factor.
[search] Added Shop POI type for ranking.
[search] Fixed category results ranking.
[search] Cleanup debug features fetching.
[search][tests] Correct affiliations initialization.
[desktop] Added "ru" locale and category checkbox.
[search] Fixed categories.
[strings] Add @501Ghost's NL translations
[android] set bottom map buttons max width to 300dp
[android] Adjust ruler and copyright to transparent nav bar
[android] Adjust compass to transparent nav bar
[android] Make navigation bar transparent on map
[android] Make splash screen navigation bar follow background color
[android] Add themed icon for Android 13
[symbols] Added norating-bg-l.png mask.
[android] Fix displaying hotel rating in search results
[styles] Add icon for archaeological_site
[android] remove custom search activity fade
[symbols] Removed background circles from icons used for tag historic and cemetery-m.svg
[generator] Fixed castle_type.
[styles] Added castle_type castrum, fortified_church, fortress, hillfort, kremlin, manor, palace & shiro
[android] Minor search wheel clean up
[drape] Fixed dropped metalines.
[styles] Increased visibility of education and some sports
[android] try to match previous map buttons layout
[android] make buttons layout react to screen height
[android] use separate buttons layout for each map mode
[android] make all map buttons independent
[android] remove main menu bar
[android] replace custom map buttons by FloatingActionButton
[android] remove layers selection from main menu
[android] improve map buttons moving behavior
[android] unify handling of map buttons
[android] simplify layers button
[strings] Add nl-Dutch translations
[ios] Use default mail app on iOS 14 to report a bug
[search] Always process viewport MWM first (after World).
[android] Note to OSM editors
[styles] Tweaks for leisure=track
[strings] Beach volleyball in English is 2 words
[strings] Fixed not working pt-BR bowling transloation for iOS
[categories] Improve german category word coverage
[bookmarks] Fixed last modified time categories order.
[altitudes] Simplify loading checks.
[string] Fix Italian generic "Stop": set "Bus stop" + "Bus tram"
[styles] Change icon of vending=fuel
[android] Increased the font size of exit number
[android] improve osm login page
[strings] update osm strings
[android] refactor profile and add changes loading indicator
[android] rework OSM login UX
[direction] Fixed some bugs for next street name
Enabled Catalan translations for iOS
Enabled Catalan TTS
[strings] Add Catalan translation
Add cross-platform system fonts in CSS for copyright.html
Add cross-platform system fonts in CSS for faq.html
[ios] Change UI font from Helvetica to system
[strings] Added missing Turkish translations
[strings] Added Telegram and Instagram links
[strings] Added additional Belarusian names of countries
Separate furniture & kitchen types
[android] Fixed possible crash.

### CODE REVIEW

File: android/src/fdroid/play/release-notes/de-DE/default.txt [hunk 1]
```diff
@@ -1,9 +1,6 @@
-• neue Kartendaten vom 16. August
-• längere Routen werden nun deutlich schneller berechnet
-• die Suchfunktion ist nun zuverlässiger und schneller
-• reduzierter Speicherplatzverbrauch der Offlinekarten
-• Fehler, dass manche Suchergebnisse nicht auf der Karte angezeigt wurden, wurde behoben
-• verbesserte Darstellung von Militärgebieten und Grenzen
-• barrierefreie Parkplätze werden nun auf der Karte angezeigt
-• verschiedene Icons für Sportarten hinzugefügt
+• Neue Kartendaten vom September
+• Neue Benutzeroberfläche für Kartenschaltflächen
```
Comment (csidirop) [inline android/src/fdroid/play/release-notes/de-DE/default.txt:2]: Better something like "Neue Tasten/Knöpfe/Buttons auf der Kartenoberfläche"

Comment (csidirop) [reply android/src/fdroid/play/release-notes/de-DE/default.txt:2]: Oh I misunderstood the English version. We changed the UI/look of the buttons, am I right (see #3398)? So maybe something like 
"Neue Oberfläche für die Tasten/Knöpfe/Buttons auf der Karte" or 
"Neues Aussehen für die Tasten/Knöpfe/Buttons auf der Karte"

**Comment by matheusgomesms** [comment]
I added PT-BR translation (#3409).

**Review by biodranik — CHANGES_REQUESTED** [review]
No no no, please wait until a properly prepared release notes PR. That one is not good.

**Comment by biodranik** [comment]
Please check updated release notes, translated with DeepL.

File: iphone/metadata/de-DE/release_notes.txt [hunk 2]
```diff
@@ -1,46 +1,32 @@
...
-• Second Hand Laden (second_hand) und Sozialkaufhaus (charity) hinzufügen
-• Hinzufügen von Tischtennis, Volleyball, Beachvolleyball, Skateboard, Schach, Padel, Handball, Futsal, Hockey, Badminton, Pelota
-• Hintergrundkreise aus den Icons entfernt
 
-Übersetzungen:
-• Arabische Übersetzungen und TTS-Texte aktualisiert
-• Fehlende Übersetzungen für place=* hinzugefügt
-• Weißrussische Übersetzungen von Ländern
-• Fehlende türkische Übersetzungen hinzugefügt
+Übersetzt mit www.DeepL.com/Translator (kostenlose Version)
```
Comment (csidirop) [inline iphone/metadata/de-DE/release_notes.txt:32]: This line should be removed

File: iphone/metadata/tr/release_notes.txt [hunk 3]
```diff
@@ -1,43 +1,32 @@
...
-• parking_entrance eklendi
-• Oyun alanı ve internet kafe simgesi eklendi
-• second_hand ve hayır kurumları eklendi
-• table_tennis, volleyball, beachvolleyball, skateboard, chess, padel, handball, futsal, hockey, badminton, pelota eklendi
-• Simgelerden arka plan daireleri kaldırıldı
 
-Çeviriler:
-• place=* için eksik çeviriler eklendi
-• Eksik Türkçe çeviriler eklendi
+Translated with www.DeepL.com/Translator (free version)
```
Comment (csidirop) [inline iphone/metadata/tr/release_notes.txt:32]: This line should be removed

**Review by MetehanOzyurek — APPROVED** [review]

File: android/src/fdroid/play/release-notes/de-DE/default.txt [hunk 4]
```diff
@@ -1,9 +1,10 @@
...
-• barrierefreie Parkplätze werden nun auf der Karte angezeigt
-• verschiedene Icons für Sportarten hinzugefügt
-meht unter omaps.app/news
+• Neue OSM-Kartendaten ab dem 12. September
+• Entfernte untere Symbolleiste für bessere UX und Sichtbarkeit der Karte
+• Verbesserte OSM-Anmeldeseite und OSM-Profil
+• Schnellere, bessere Suche
+• Absturz beim Starten nach der Bearbeitung behoben
+• Lesezeichenlisten sind nun nach der letzten Änderungszeit sortiert
+• Fixierte Übersetzungen
```
Comment (j5k) [inline android/src/fdroid/play/release-notes/de-DE/default.txt:7]: Verbesserte Übersetzungen ('Fixierte' is nonsense in this way)

**Comment by biodranik** [comment]
@csidirop should we wait until you improve the German translation?

**Comment by csidirop** [comment]
> @csidirop should we wait until you improve the German translation?

Sorry had no time yesterday. I will look over it in my break time. In about two hours ok?

File: android/src/fdroid/play/release-notes/de-DE/default.txt [hunk 5]
```diff
@@ -1,9 +1,10 @@
...
-• längere Routen werden nun deutlich schneller berechnet
-• die Suchfunktion ist nun zuverlässiger und schneller
-• reduzierter Speicherplatzverbrauch der Offlinekarten
-• Fehler, dass manche Suchergebnisse nicht auf der Karte angezeigt wurden, wurde behoben
-• verbesserte Darstellung von Militärgebieten und Grenzen
-• barrierefreie Parkplätze werden nun auf der Karte angezeigt
-• verschiedene Icons für Sportarten hinzugefügt
-meht unter omaps.app/news
+• Neue OSM-Kartendaten ab dem 12. September
+• Entfernte untere Symbolleiste für bessere UX und Sichtbarkeit der Karte
```
Comment (csidirop) [inline android/src/fdroid/play/release-notes/de-DE/default.txt:2]: "Entfernte untere Symbolleiste ..." sounds somehow wrong for me. Use "Untere Symbolleiste entfernt ..." instead.

**Review by csidirop — APPROVED** [review]

**Review by csidirop — APPROVED** [review]

File: iphone/metadata/en-US/release_notes.txt [hunk 6]
```diff
@@ -1,46 +1,29 @@
...
-• Add table_tennis, volleyball, beachvolleyball, skateboard, chess, padel, handball, futsal, hockey, badminton, pelota
-• Removed background circles from icons
-
-Translations:
-• Updated Arabic translations and TTS texts
-• Added missing translations for place=*
-• Belarusian translations of countries
-• Added missing Turkish translations
+Localizations:
+• Added missing translations for craft=*
```
Comment (biodranik) [inline iphone/metadata/en-US/release_notes.txt:26]:
```suggestion
• Added missing translations for craft=* and residential gardens
```

**Review by biodranik — APPROVED** [review]

**Review by matheusgomesms — APPROVED** [review]

**Pull request merged**